Cargo.toml

```diff
@@ -10,6 +10,15 @@ members = [
 ]
 resolver = "2"
 
+[workspace.package]
+version = "0.8.1"
+authors = ["thin-edge.io team <info@thin-edge.io>"]
+edition = "2021"
+rust-version = "1.64"
+license = "Apache-2.0"
+homepage = "https://thin-edge.io"
+repository = "https://github.com/thin-edge/thin-edge.io"
```

Comment:
We define all this common metadata here and inherit them from here in individual crate's Cargo.toml